common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -101,6 +107,9 @@ public class CommonValue {
             + MLModel.CREATED_TIME_FIELD
             + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
             + "      \""
+            + MLModel.LAST_UPDATE_TIME_FIELD
```

Comment:
with keeping the similar convention with `CREATED_TIME_FIELD` and `LAST_UPLOADED_TIME_FIELD` may be we can rename to : `LAST_UPDATED_TIME_FIELD`

Reply:
This is to keep consistent with MLTask.LAST_UPDATE_TIME_FIELD. I think it's reasonable to keep consistent with other fields of ML model. Will change this part.